server/command.go

```diff
@@ -17,6 +17,7 @@ const helpText = "###### Mattermost Jira Plugin - Slash Command Help\n" +
 	"* `/jira create <text (optional)>` - Create a new Issue with 'text' inserted into the description field\n" +
 	"* `/jira transition <issue-key> <state>` - Change the state of a Jira issue\n" +
 	"* `/jira view <issue-key>` or `/jira <issue-key>` - View a Jira issue\n" +
+	"* `/jira subscribe` - Create or edit a subscription for this channel\n" +
```

Comment:
@aaronrothschild suggest: `Create or edit a Jira webhook subscription for this channel`?

Reply:
```suggestion
	"* `/jira subscribe` - Configure the Jira notifications sent to this channel \n" +
```
Since it's opening a modal, we don't need any commands.

Reply:
Hmmm, shouldn't then the command be `/jira notify`? I know... I know...